github-connector/routing_test.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/KarolJaksik/hack-showcase/github-connector/mocks"
+	"github.com/google/go-github/github"
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type payload struct {
+	Anything string `json:"anything"`
+}
+
+//createRequest creates an HTTP request for test purposes
+func createRequest(t *testing.T) (*httptest.ResponseRecorder, *http.Request, *mocks.Validator, []byte) {
```

Comment:
This is a lot of return values, consider separating creation of handler from request and response recorder (this one can be done in the test case itself).